GTG/core/base_store.py

```diff
@@ -25,17 +25,52 @@
 import logging
 
 from lxml.etree import _Element
-from typing import Dict, List, Optional, TypeVar, Generic, Protocol
+from typing import Dict, List, Optional, TypeVar, Generic
+from typing_extensions import Self
 
 
 log = logging.getLogger(__name__)
 
-S = TypeVar('S',bound='Storeable')
+S = TypeVar('S',bound='StoreItem')
+
+class StoreItem(GObject.Object):
+    """Base class for items in BaseStore."""
+
+    __gtype_name__ = 'gtg_StoreItem'
+
+
+    def __init__(self,id: UUID):
+        self.id: UUID = id
+        self.parent: Optional[Self] = None
+        self.children: List[Self] = []
+        super(StoreItem, self).__init__()
```

Comment:
```suggestion
        super().__init__()
```
Bad habits form Python 2 still in the code base :) Let's fix them as we see them.

Reply:
Done.